src/opm/io/eclipse/EclFile.cpp

```diff
@@ -158,8 +158,8 @@ unsigned long int sizeOnDiskBinary(long int num, Opm::EclIO::eclArrType arrType)
             int maxBlockSize = std::get<1>(sizeData);
             int maxNumberOfElements = maxBlockSize / sizeOfElement;
 
-            long unsigned int numBlocks = static_cast<unsigned long int>(num)/static_cast<unsigned long int>(maxNumberOfElements);
-            long unsigned int rest = static_cast<unsigned long int>(num) - numBlocks*static_cast<unsigned long int>(maxNumberOfElements);
+            long unsigned int numBlocks = static_cast<uint64_t>(num)/static_cast<uint64_t>(maxNumberOfElements);
+            long unsigned int rest = static_cast<uint64_t>(num) - numBlocks*static_cast<uint64_t>(maxNumberOfElements);
```

Comment:
Here I would use `auto numBlocks = `

Reply:
Thanks, sorry that I missed a few there. Updated now.